provider/keycloak_user.go

```diff
@@ -49,6 +49,30 @@ func resourceKeycloakUser() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"attributes": {
+				Type:     schema.TypeMap,
+				Optional: true,
+			},
+			"federated_identity": {
```

Comment:
Should this be `Computed`?  My understanding is that this only exists if the user was federated from an identity provider.